yogstation/code/modules/donor/donor_items.dm

```diff
@@ -70,3 +70,40 @@ var/list/donor_start_items = list(\
 						)
 
 var/list/donor_pdas = list("Normal", "Transparent", "Pip Boy", "Rainbow")
+
+var/list/donor_start_tools = list(\
+						/obj/item/banhammer, \
+						/obj/item/bedsheet/wiz, \
+						/obj/item/bedsheet/nanotrasen, \
+						/obj/item/bedsheet/syndie, \
+						/obj/item/bikehorn/airhorn, \
+						/obj/item/camera, \
+						/obj/item/cane, \
+						/obj/item/clothing/shoes/sneakers/rainbow, \
+						/obj/item/clothing/gloves/color/rainbow, \
+						/obj/item/clothing/under/color/rainbow, \
+						/obj/item/grown/sunflower, \
+						/obj/item/hot_potato/harmless/toy, \
+						/obj/item/instrument/accordion, \
+						/obj/item/instrument/glockenspiel, \
+						/obj/item/instrument/harmonica, \
+						/obj/item/instrument/recorder, \
+						/obj/item/instrument/saxophone, \
+						/obj/item/latexballon, \
+						/obj/item/lighter, \
+						/obj/item/lipstick/random, \
+						/obj/item/poster/random_contraband, \
+						/obj/item/poster/random_official, \
+						/obj/item/razor, \
+						/obj/item/toy/balloon, \
+						/obj/item/toy/beach_ball, \
+						/obj/item/toy/clockwork_watch, \
+						/obj/item/toy/dummy, \
+						/obj/item/toy/eightball, \
+						/obj/item/toy/katana, \
+						/obj/item/toy/gun, \
+						/obj/item/toy/plush/plushvar, \
+						/obj/item/toy/plush/narplush, \
+						/obj/item/toy/sword, \
+						/obj/item/gun/magic/wand
```

Comment:
No

Reply:
lol

Reply:
yeah wtf why use /‘s when you dont need to

Reply:
backslashes is how we already do them for hats and I enjoy consistency. complain to Ross if you don't like the items we have since its his list.

Reply:
Remove the magic wand

Reply:
no u